Description: Use main version instead of git version
Author: Stephen Kitt <skitt@debian.org>

This avoids needing git to build.

--- a/meson.build
+++ b/meson.build
@@ -6,10 +6,6 @@
 )
 
 git_version = []
-git = find_program('git', required: false)
-if git.found()
-  git_version = run_command(git, 'describe', '--abbrev=4', '--dirty').stdout().strip().split('-')
-endif
 
 # libtool versioning
 lt_current = 1
